src/main/redux/actions/win/registry/registerReaderPublication.ts

```diff
@@ -7,27 +7,19 @@
 
 import { Rectangle } from "electron";
 import { Action } from "readium-desktop/common/models/redux";
-import { IReaderStateReader } from "readium-desktop/common/redux/states/renderer/readerRootState";
+import { IReaderStateReaderPersistence } from "readium-desktop/common/redux/states/renderer/readerRootState";
 
 export const ID = "WIN_REGISTRY_REGISTER_READER_PUBLICATION";
 
 export interface Payload {
     publicationIdentifier: string;
     bound: Rectangle;
-    reduxStateReader: IReaderStateReader;
+    reduxStateReader: Partial<IReaderStateReaderPersistence>;
 }
 
-export function build(publicationIdentifier: string, bound: Rectangle, reduxStateReader: IReaderStateReader):
+export function build(publicationIdentifier: string, bound: Rectangle, reduxStateReader: Partial<IReaderStateReaderPersistence>):
     Action<typeof ID, Payload> {
 
-    // https://github.com/edrlab/thorium-reader/issues/1444
-    // TODO:
-    // see initStore() in main/redux/store/memory.ts
-    // !!
-    delete reduxStateReader.highlight;
-    // https://github.com/edrlab/thorium-reader/blob/fd9d82beb95d5d2ee4c927a2eed91ebf3ffa8424/src/main/redux/actions/win/session/registerReader.ts#L57
-    delete reduxStateReader.info;
```

Comment:
`delete reduxStateReader.info;` is not necessary now, thanks to : 
https://github.com/edrlab/thorium-reader/pull/2946/files#diff-7de918470e1f77df422151ca5c658e410828cc273ad1b7abfc91a77d38251c8eR141-R145

do not update readerStatePersistence on reader closing, the state must not be updated from readerStateSession